src/cfml/system/modules_app/server-commands/commands/server/open.cfc

```diff
@@ -47,8 +47,8 @@ component {
 				arguments.URI = '/' & arguments.URI;
 			}
 			var thisURL = "#serverInfo.host#:#serverInfo.port##arguments.URI#";
-			print.greenLine( "Opening...#thisURL#" );
-			openURL( thisURL );
+			print.greenLine( "Opening...#thisURL# on #serverInfo.preferedBrowser#" );
```

Comment:
This setting will be an empty string most of the time, you can't count on it here.  Furthermore, I don't think the default browser should even be set at a server level.  This is just a top config setting for all of CommandBox.